ush/config_defaults.yaml

```diff
@@ -2811,63 +2107,27 @@ verification:
   OBS_NDAS_SFCorUPA_FN_TEMPLATE: 'prepbufr.ndas.{valid?fmt=%Y%m%d%H}'
   OBS_NDAS_SFCorUPA_FN_METPROC_TEMPLATE: '${OBS_NDAS_SFCorUPA_FN_TEMPLATE}.nc'
   #
-  # VX_LOCAL_MODULE_FN:
-  # Name (without extension) of the local module file for running the vx
-  # tasks in the workflow.
-  #
-  VX_LOCAL_MODULE_FN: 'run_vx'
-  #
-  # RUN_TASKS_METVX_DET:
-  # Flag that specifies whether to run deterministic verification.  If set
-  # to True, this will run deterministic vx on the post-processed forecast
-  # output.  This post-processed output may consist of a single forecast
-  # or an ensemble of foreasts, and it may be staged from previous runs of
-  # the SRW App or may be generated by running the TN_RUN_FCST task as part
-  # of the current SRW-App-generated experiment.
-  #
-  # RUN_TASKS_METVX_ENS:
-  # Flag that specifies whether to run ensemble verification.  The ensemble
-  # forecast output on which vx will be run may be staged or generated by
-  # running an ensemble of forecasts with the weather model as part of the
-  # current SRW-App-generated experiment.
-  #
-  RUN_TASKS_METVX_DET: False
-  RUN_TASKS_METVX_ENS: False
-  #
   # VX_FCST_MODEL_NAME:
   # String that specifies a descriptive name for the model being verified.
   # This is used in forming the names of the verification output files as
   # well as in the contents of those files.
   #
-  # VX_FIELDS:
-  # The fields or groups of fields on which to run verification.
-  #
-  # VX_APCP_ACCUMS_HH:
-  # The 2-digit accumulation periods (in units of hours) to consider for
-  # APCP (accumulated precipitation).  If VX_FIELDS contains "APCP", then
-  # VX_APCP_ACCUMS_HH must contain at least one element.  If not,
-  # VX_APCP_ACCUMS_HH will be ignored.
-  #
   VX_FCST_MODEL_NAME: '{{ nco.NET }}.{{ task_run_post.POST_OUTPUT_DOMAIN_NAME }}'
-  VX_FIELDS: [ "APCP", "REFC", "RETOP", "SFC", "UPA" ]
```

Comment:
@mkavulich If I want to run vx on just a subset of these fields, how is that now done?